_dashboards-assistant/index.md

```diff
@@ -0,0 +1,100 @@
+Under the response, we also provide suggestions that we think user may be interested to ask. User can click one of them to see the response.
+
+<img width="600px" src="{{site.url}}{{site.baseurl}}/images/dashboards-assistant/suggestions.png" alt="The assistant suggestions">
+
+### Rate the response
+Each response contains a thumb up and thumb down icon button, you can click on the button to feedback whether you are satisfied with the response.
+
+<img width="600px" src="{{site.url}}{{site.baseurl}}/images/dashboards-assistant/rate.png" alt="Rate the response">
+
+### See how the response was generated
+Under the available suggestions, there will be a link “How was this generated?”, click this link will show which tools are triggered to generate this response. It can be used for trouble shooting. There could be multiple steps if multiple tools are involved. Each step will display tool name with input and output of the tool.
+
+<img width="400px" src="{{site.url}}{{site.baseurl}}/images/dashboards-assistant/traces.png" alt="How was the response generated">
+
+### Continue with a past conversation
+#### View past conversation
+Click the clock icon to open the conversation history panel. The clock icon will become active status and conversations will be displayed clicked after clicked.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards-assistant/conversation-history-entry.png" alt="Show conversation history">
+
+Search conversations by title:
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards-assistant/conversation-history-list.png" alt="List of conversations">
+
+#### Manage past conversation
+Click the pencil icon to edit conversation name, type the new conversation name in the below input, then click “Confirm name” button. The new conversation name will be saved.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards-assistant/edit-conversation-title.png" alt="Edit the conversation title">
+
+Click the trash icon to delete conversation, the delete confirmation modal will be displayed after clicked. Click the “Delete conversation” button to confirm the conversation delete operation. The conversation will be deleted after “Delete conversation” button clicked.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards-assistant/delete-conversation.png" alt="Delete the conversation">
+
+### Share a conversation via notebook
+Click `Save to notebook` button to save your conversation to notebook. All messages between you and LLM model will be saved to observability notebook after entering name for notebook and confirm.
+
+<img width="400px" src="{{site.url}}{{site.baseurl}}/images/dashboards-assistant/save-conversation-to-notebook.png" alt="Save a conversation to notebook">
+
+Now you can find your conversation in observability notebook lists and others can see it as well.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards-assistant/conversation-in-notebook.png" alt="Conversation saved to notebook">
```

Comment:
```suggestion
<img src="{{site.url}}{{site.baseurl}}/images/dashboards-assistant/conversation-in-notebook.png" alt="List of conversations saved in Notebooks">
```

Reply:
Add new section 

## Relate information

The following resources provide more information about the OpenSearch Assistant Toolkit and related components:

- [OpenSearch adds a new generative AI assistant toolkit](https://opensearch.org/blog/opensearch-adds-new-generative-ai-assistant-toolkit/)
- <other resources>